lib/active_shipping/carriers/ups.rb

```diff
@@ -816,16 +816,18 @@ def parse_tracking_response(response, options = {})
         # Build status hash
         status_nodes = first_package.css('Activity > Status > StatusType')
 
-        # Prefer a delivery node
-        status_node = status_nodes.detect { |x| x.at('Code').text == 'D' }
-        status_node ||= status_nodes.first
+        if status_nodes.present?
```

Comment:
If defined within a conditional, it will be `nil`, not `undefined`, which means the check on line 838 will be `unless nil == :delivered`.

Is that acceptable, or should we still set some kind of status?